coverage/dist/app/routes/googleLogin.js.html

```diff
@@ -0,0 +1,206 @@
+&nbsp;
+var _express = require('express');
+&nbsp;
+var _express2 = _interopRequireDefault(_express);
+&nbsp;
+var _firebase = require('firebase');
+&nbsp;
+var _firebase2 = _interopRequireDefault(_firebase);
+&nbsp;
+var _db = require('../../config/db');
+&nbsp;
+function _interopRequireDefault(obj) { return obj &amp;&amp; obj.__esModule ? obj : { default: obj }; }
+&nbsp;
+var app = (0, _express2.default)(); // google route
+// using firebase authentication method
+&nbsp;
+var googleLogin = function googleLogin(app) {
+  app.post('/user/google', <span class="fstat-no" title="function not covered" >fu</span>nction (req, res) {
+    var token = <span class="cstat-no" title="statement not covered" >void 0,</span>
+        email = <span class="cstat-no" title="statement not covered" >void 0,</span>
+        uid = <span class="cstat-no" title="statement not covered" >void 0,</span>
+        displayName = <span class="cstat-no" title="statement not covered" >void 0;</span>
+    var provider = <span class="cstat-no" title="statement not covered" >new _firebase2.default.auth.GoogleAuthProvider();</span>
+<span class="cstat-no" title="statement not covered" >    provider.addScope('https://www.googleapis.com/auth/plus.login');</span>
+<span class="cstat-no" title="statement not covered" >    (0, _db.firebaseAuth)().signInWithPopup(provider).then(<span class="fstat-no" title="function not covered" >fu</span>nction (result) {</span>
+<span class="cstat-no" title="statement not covered" >      token = result.credential.accessToken;</span>
+<span class="cstat-no" title="statement not covered" >      email = result.user.email;</span>
+<span class="cstat-no" title="statement not covered" >      uid = result.user.uid;</span>
+<span class="cstat-no" title="statement not covered" >      displayName = result.user.displayName;</span>
+    }).then(<span class="fstat-no" title="function not covered" >fu</span>nction (snap) {
+      var userRef = <span class="cstat-no" title="statement not covered" >_firebase2.default.database().ref('users/').child(uid).set({</span>
+        username: displayName,
+        email: email
+      });
+    }).catch(<span class="fstat-no" title="function not covered" >fu</span>nction (error) {
+<span class="cstat-no" title="statement not covered" >      res.send({</span>
+        message: error.message
+      });
+    });
+  });
+};
+&nbsp;
+exports.default = googleLogin;</pre></td></tr>
+</table></pre>
+<div class='push'></div><!-- for sticky footer -->
+</div><!-- /wrapper -->
+<div class='footer quiet pad2 space-top1 center small'>
+  Code coverage
+  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon Sep 11 2017 11:34:29 GMT+0100 (WAT)
+</div>
+</div>
+<script src="../../../prettify.js"></script>
+<script>
+window.onload = function () {
+        if (typeof prettyPrint === 'function') {
+            prettyPrint();
+        }
+};
+</script>
+<script src="../../../sorter.js"></script>
+</body>
+</html>
```

Comment:
Hi @sasili-adetunji. I'm sure this generated files under coverage should be git ignored. they shouldn't be pushed online. Try to ignore all files that need ignoring